bin/jobsub_submit

```diff
@@ -211,6 +216,13 @@ def main():
     backslash_escape_layer(sys.argv)
     args = parser.parse_args()
 
+    # Allow environment variables to append to some command lists to get rid of
+    # need for poms_jobsub_wrapper to get in front of us on the path -- we will
+    # just set these and have a job-info script report the job id, etc.
+    args.environment.extend(get_env_list("JOBSUB_EXTRA_ENVIRONMENT"))
+    args.lines.extend(get_env_list("JOBSUB_EXTRA_LINES"))
```

Comment:
Do we want a sanitizer (maybe in utils.py) so that a user can't accidentally throw junk into the submit file?

Reply:
I would be okay with that, but what would it sanitize?  I mean, we should probably be doing that to --lines also... would we have a regexp that matches classad definition lines, etc?

Reply:
I found the Right Answer -- we have Condor check it, with https://htcondor.readthedocs.io/en/latest/apis/python-bindings/api/classad.html#classad.parseOne

Reply:
Love it.